.storybook/components/ColorSwatches/ColorSwatches.module.css

```diff
@@ -1,28 +1,19 @@
-.colorbar {
-  display: flex;
-  width: 100%;
-}
-
-.color {
-  flex: 1;
-  text-align: center;
+:root {
+  --swatchSize: 64px;
```

Comment:
[Size  ≠ space](https://atlantis.getjobber.com/?path=/docs/guides-frontend-styleguide--docs#size-width-and-height)